src/utils/Membership.js

```diff
@@ -0,0 +1,53 @@
+const onKeyPress = event => {
+  const keyCode = event.keyCode || event.which;
+  const keyValue = String.fromCharCode(keyCode);
+  if (/\+|-|e/.test(keyValue)) event.preventDefault();
+};
```

Comment:
What is this function doing? Can we add some comments?

Reply:
Done